### DISCUSSION TITLE
Move the changelog.md

### DISCUSSION THREAD

**Discussion by togfoxy** [togfoxy/MarsLander#259] [Ideas]
Do you think we are able to move changelog.md into the source folder? It would help my workflow tremendously because when I make changes I upload all of source to github but this doesn't include changelog.md which is in the parent folder. I then have to make a second action to edit and upload the change log.

I think having the changelog in the source folder would increase the chances of me updating and uploading it to github. Would that upset the workflow you have?

**Comment by Philbywhizz** [comment]
I'd be inclined to leave it in the root of the folder to be honest. It's not source code, its documentation for the whole project. It is where people expect it to be located. If you bury it in a source folder, people don't know where to find it.

**Comment by togfoxy** [comment]
They are change notes for us. Players don't see them.

It's okay - I've install ATOM which fixes it anyway.

**Reply by MadByteDE** [reply]
I'd also suggest to use something like Gitkraken which makes everythin git related a lot easier.. at least for me.
